x-pack/plugins/security_solution/cypress/tasks/common.ts

```diff
@@ -280,3 +281,8 @@ export const waitForWelcomePanelToBeLoaded = () => {
   cy.get(KIBANA_LOADING_ICON).should('exist');
   cy.get(KIBANA_LOADING_ICON).should('not.exist');
 };
+
+export const waitForTableToLoad = () => {
+  cy.get(EUI_BASIC_TABLE_LOADING).should('exist');
```

Comment:
The only thing I'm not quite sure is about checking for the loading icon to know if the table was successfully loaded or not. I've seen some places where doing that has introduced flakiness at the end, there is no other way or element to check that the table was successfully loaded?

Reply:
The only visual indicator that a `EuiBasicTable` is loading is the blue bar animation at the table header. The animations appear when the table has the `EUI_BASIC_TABLE_LOADING` class name. From what I know, that is the only way to verify if a `EuiBasicTable` is loading.

<img width="856" alt="Screenshot 2023-07-18 at 12 43 49" src="https://github.com/elastic/kibana/assets/1490444/782151ea-d957-485c-847e-7a546c0df9a0">